src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
+            placeholder_alert_view.title = _("No new firmware available");
```

Comment:
This is not necessarily true. There may be new firmware for your device, it's just not available in fwupd

Reply:
Fair point, but the distinction isn't there between new and old.

Reply:
```suggestion
            placeholder_alert_view.title = _("No new firmware found");
```